modules/references/server/controllers/reference.server.controller.js

```diff
@@ -72,8 +73,23 @@ function validateCreate(req) {
     details.userTo = 'userId expected';
   }
 
+  if (_.has(req, ['body', 'feedbackPublic'])) {
+    const feedbackPublic = req.body.feedbackPublic;
```

Comment:
What kind of sanitation we should add here?

Reply:
See https://github.com/Trustroots/trustroots/pull/1385/files#r409637173

Reply:
(I split the server change off into a seperate branch/PR)